code/game/objects/items/devices/paicard.dm

```diff
@@ -52,6 +52,8 @@
 		else
 			dat += "<b>Radio Uplink</b><br>"
 			dat += "<font color=red><i>Radio firmware not loaded. Please install a pAI personality to load firmware.</i></font><br>"
+		if(pai.aiPDA.id)
+			dat += "<A href='byond://?src=[REF(src)];ejectid=1'>\[Eject ID\]</a><br>"
```

Comment:
did you test it
```suggestion
			dat += "<a href='byond://?src=[REF(src)];ejectid=1'>[Eject ID]</a><br>"
```

Reply:
...I am 90% sure this would break things.

Reply:
It works just fine as is.